src/index.html

```diff
@@ -13,7 +13,7 @@
       <div class="logo"><h1>My <span>Meals</span></h1></div>
       <nav>
         <ul>
-          <li id="meals">Meals</li>
+          <li id="meals"></li>
```

Comment:
Thank you for your commit. I propose to keep  this menu option for future development. We can add few pages and navigate to Categories, Ingridients and back to Meals.